index.d.ts

```diff
@@ -410,6 +531,25 @@ declare module 'warframe-items' {
         levelStats: LevelStat[];
     }
 
+    interface TitaniaRelicLocation {
+        location: string;
+        rarity: Rarity;
+        chance: number;
+    }
+    interface TitaniaRelicReward {
+        rarity: Rarity;
+        chance: number;
+        item: TitaniaRelicRewardItem;
+    }
+    interface TitaniaRelicRewardItem {
+        name: string;
+        warframeMarket: TitaniaWFMInfo | null;
```

Comment:
This would also be for future proffing? Because all relics have a warframe market link.

Reply:
ah, unfortunately Forma (any TitaniaRelicRewardItem w/ name: 'Forma') do not.

Reply:
So far all items that contain "Forma", "Kuva", "Exilus" or "Riven" won't have any wfm item info, since they aren't tradable afaik. But I can see how that is more annoying to work with. I could add them later today if you want

Reply:
Ultimately, as long as they're marked nullable, i dunno that it's a problem? It'd be nice if they weren't, but that's just reality, lol

Reply:
Its not a problem per say, just one more null check, but otherwise the id and url would be null i guess 😄

Reply:
Yeah, hopefully people are wisely using existence chaining.

Reply:
Actually it's the relic warframe market data, i mistook by its reward. Having a null or undefined to represent an optional value is totally fine.